### ISSUE TITLE
Big picture mode

### ISSUE THREAD

**Issue by AJV20** [Phalcode/gamevault-app#208]
**Is your feature request related to a problem? Please describe.**
Not a problem more like a nice to have feature

**Describe the solution you'd like**
Would be great if we could have a big picture mode similar to Steam or Playnite. Sometimes I play games on my ROG Ally and it makes it more of a console experience. 

It would also be helpful when used with only a controller. Would be even better if there was an option to automatically open the app in big picture mode to avoid having to use a keyboard and mouse. So it would turn on the pc and all that would be needed is a controller. 

**Describe alternatives you've considered**
Adding controller support and a setting to open the app automatically could be a less complicated solution

**Comment by Yelo420** [comment]
Interesting idea, we will discuss this internally.
The autostart setting already exists, but it is not automatically activated.

**Comment by yodatak** [comment]
Steam deck users will be interested also in Linux, there is more steam deck and Linux user in steam than in macos by exemple

**Comment by Alfagun74** [comment]
Absolutely, this would definitely be a great idea for folks using the Steam Deck, or for those like me who stream GameVault to their Smart TV using Moonlight.

**Comment by Alfagun74** [comment]
While I fully understand your desire for a big picture mode, I would suggest limiting its functionality to only launching games that are already installed.   
  
This means excluding features such as settings, community or installing new games.   
  
Adding these additional features would essentially require developing a whole second user interface and introduce complexities like controller compatibility.   
   
We will need to discuss this further, but I can assure you that even with this limited set of features, it is something for later and not in our main focus as of right now.

**Comment by Alfagun74** [comment]
We would definitely look closer into this if we should rewrite the client cross-platform
